Mathlib/Topology/Algebra/SeparationQuotient/Basic.lean

```diff
@@ -379,6 +386,19 @@ def mkCLM : M →L[R] SeparationQuotient M where
   map_add' := mk_add
   map_smul' := mk_smul
 
+variable {R M}
+
+/-- The lift as a continuous linear map of `f` with `f x = f y` for `Inseparable x y`. -/
```

Comment:
```suggestion
/-- The lift (as a continuous linear map) of `f` with `f x = f y` for `Inseparable x y`. -/
```
to avoid thinking it's a continuous linear map of `f`